### PR TITLE
[DH-274] Changing shared-readwrite from /opt to /home/jovyan in Stat 20 hub for infra admins!

### PR DESCRIPTION
**Pull request by balajialg**
Okay, I just changed the stanzas to mimic Stat 159 hub. Let me know if you are okay with it?

Re: https://github.com/berkeley-dsep-infra/datahub-usage-analysis/pull/17
https://jira-secure.berkeley.edu/browse/DH-274

### CODE REVIEW

**Review by ryanlovett — CHANGES_REQUESTED** [review]
First, sorry for not mentioning this earlier in the other thread where I referenced stat159 and /opt...

Second, I think we shouldn't alter the existing volume mounts -- we should add a new one just for this project. The existing ones are for course data. If you were to merge as is, and drop the data into `_shared`, the students would be able to see the data.

- This should be a new volume mount scoped under `group_profiles.course::1524699::group::all-admins:'
- It shouldn't use the directory name `shared-readwrite` since 1) the volume will be mounted read-only and 2) `shared-readwrite` and `shared` are part of a convention used in the hubs across all courses. That name should be set aside for course data. Since this change is for datahub admins, we can mount it as `/home/jovyan/shared-524699-admins` or some such.
- `_shared` should be `_shared-524699-admins` or some such since `_shared` is again reserved for course data.

**Comment by balajialg** [comment]
Thanks @ryanlovett for the clarification. I was aware of the protocol for shared/shared-read-write directories since I had handled [such requests in the past](https://github.com/berkeley-dsep-infra/datahub/pull/5644/commits/31f2199e52b57bc0c9a6f93293d5e18cbbdafabd). However, I got confused with the admin stanza listed below. For a moment, I thought that this stanza needs to get changed for admins instead of adding a new stanza linking bcourses id (as we do for user requests). My bad! 

```
    admin:
      extraVolumeMounts:
        - name: home
          mountPath: /opt/shared-readwrite
          subPath: _shared
```

**Comment by balajialg** [comment]
@ryanlovett @shaneknapp Let me know if you are okay with merging the changes to staging?

File: deployments/stat20/config/common.yaml [hunk 1]
```diff
@@ -86,8 +86,14 @@ jupyterhub:
...
+        extraVolumeMounts:
+          - name: home
+            mountPath: /home/jovyan/admins-shared-storage
+            subPath: _shared/admins-storage
     admin:
       extraVolumeMounts:
         - name: home
           mountPath: /opt/shared-readwrite
-          subPath: _shared
+          subPath: _shared
```
Comment (shaneknapp) [inline deployments/stat20/config/common.yaml]: add a new line after `_shared` please

Comment (balajialg) [reply deployments/stat20/config/common.yaml]: Done

**Comment by shaneknapp** [comment]
looks good to me...

**Pull request merged**